code/modules/paperwork/photography.dm

```diff
@@ -150,6 +150,22 @@
 	var/icon_on = "camera"
 	var/icon_off = "camera_off"
 	var/size = 3
+	var/see_ghosts = 0 //for the spoop of it
+
+
+/obj/item/device/camera/CheckParts()

```

Comment:
This code should probably be in `/obj/item/device/camera/spooky/CheckParts()` instead, to avoid any future table-crafted camera subtypes displaying this message.